docs/JLBP-0005.md

```diff
@@ -8,6 +8,11 @@ permalink: /JLBP-5
 Definition: When the same fully qualified class name is provided by
   two or more distinct artifacts (different group IDs, different
   artifact IDs, or both) the classes are said to *overlap*.
+  
+When a class appears in more than one location in the classpath, it is unpredictable which version
```

Comment:
```suggestion
When a class appears in more than one location in the class path, it is unpredictable which version
```
https://docs.oracle.com/javase/8/docs/technotes/tools/windows/classpath.html

Reply:
1. Java is Oracle's technology. Not O’Reilly's.
1. It's more up-to-date to call it "class path". Java 8 referenced the option as "-classpath". Newer distribution Java 11 references the option "-class-path". 

(old)
```
-classpath classpath1:classpath2...
```
Java 8 https://docs.oracle.com/javase/8/docs/technotes/tools/unix/classpath.html#CBHHCGFB

(new)
```
--class-path classpath, -classpath classpath, or -cp classpath
A semicolon (;) separated list of directories, JAR archives, and ZIP archives to search for class files.

Specifying classpath overrides any setting of the CLASSPATH environment variable. If the class path option isn’t used and classpath isn’t set, then the user class path consists of the current directory (.).
```
Java 11 https://docs.oracle.com/en/java/javase/11/tools/java.html#GUID-3B1CE181-CD30-4178-9602-230B800D4FAE

Reply:
done